std/json.d

```diff
@@ -871,9 +886,13 @@ struct JSONValue
     ///
     @safe unittest
     {
-        assert(JSONValue(0u) == JSONValue(0));
-        assert(JSONValue(0u) == JSONValue(0.0));
-        assert(JSONValue(0) == JSONValue(0.0));
+        assert(JSONValue(10).opEquals(JSONValue(10.0)));
```

Comment:
If I use `==` here, the generator will convert it to:

```
writeln(JSONValue(10)); // JSONValue(10.0)
```

Hence I'm calling `opEquals` explicitly here.

Reply:
I recall such awkward situations coming up before. Maybe it's time to add some kind of `// no-writeln` syntax to https://github.com/dlang/dlang.org/blob/master/ddoc/source/assert_writeln_magic.d ?

Reply:
https://github.com/dlang/dlang.org/pull/3801

Reply:
Or it could exempt documented unittests under `opEquals`